Tooling/ScriptGenerator/Services/RemoveInheritCodeDocsMutation.cs

```diff
@@ -7,7 +7,7 @@
 namespace ScriptGenerator.Services;
 
 /// <summary>
-/// Removes any test lines from a <c>string</c> that contains inherit XML documentation code.
+/// Removes any test lines from a <c>string</c> that contain inherit XML documentation code.
```

Comment:
You also need to change inherit to inherent
```suggestion
/// Removes any test lines from a <c>string</c> that contain inherent XML documentation code.
```